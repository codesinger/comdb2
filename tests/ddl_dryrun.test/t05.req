dryrun create table t3 { schema { int a } keys { "foo" = foo } }
