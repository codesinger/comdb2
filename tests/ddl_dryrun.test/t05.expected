[dryrun create table t3 { schema { int a } keys { "foo" = foo } }] failed with rc 240 Error at line   0: SYMBOL NOT FOUND: foo.
